config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
+    resources :invoices
```

Comment:
This is fixed in the other PR with the routes, don't know if you want to leave it there

Reply:
Is this where the failed test is coming from?